lib/IHP/DataSync/ihp-querybuilder.js

```diff
@@ -82,4 +82,18 @@ function query(table) {
     return new QueryBuilder(table);
 }
 
+function jsValueToDynamicValue(value) {
+    if (typeof value === "string") {
+        return { tag: 'TextValue', contents: value };
+    } else if (typeof value === "number") {
+        return { tag: 'IntValue', contents: value };
+    } else if (typeof value === "boolean") {
+        return { tag: 'BoolValue', contents: value };
+    } else if (value === null || value === undefined) {
+        return { tag: 'NullValue' };
+    }
+
+    throw new Error('Could no transform JS value to DynamicValue');
```

Comment:
Maybe indicate which values are valid?

```suggestion
    throw new Error('Could no transform JS value to DynamicValue (Text, Int, Bool)');
```

Reply:
btw, do we support Float?

Reply:
Good point 👍 will add this.

Floats are not yet implemented

Reply:
Created an issue for Floats here https://github.com/digitallyinduced/ihp/issues/1298